extension/content_script.js

```diff
@@ -440,13 +469,13 @@ function setupKeyListeners() {
 				}
 				break;
 			case 'ShiftRight':
-				persistentHighlight();
+				togglePersistentHighlight();
 				break;
 			case 'ShiftLeft':
 				persistentHighlight();
 				break;
 			case 'Slash':
-				toggleKeywordSettings();
+				togglePersistentHighlight();
```

Comment:
Whoops! I think this was meant to go under ShiftLeft instead of Slash; 

Other than this, feel free to merge afterward

Reply:
OOOOPS. Thanks for catching it!